### DISCUSSION TITLE
How to run the kubernetes client system on local machine

### DISCUSSION THREAD

**Discussion by Ananya2001-an** [fabric8io/kubernetes-client#4262] [Q&A] [answered]
Hey! 
I am new to open source and I am interested in contributing to this repository.  But for that I want to setup the client system on my local machine first so that I can run some tests and understand how everything is working. 
Can someone guide me with the steps as to how we can run this client on local host?
I would love a detailed explanation.

**Comment by rohanKanojia** [comment, marked as answer]
Have you checked out our [Contributing guide](https://github.com/fabric8io/kubernetes-client/blob/master/CONTRIBUTING.md)? You would need to set up some local Kubernetes Cluster (kind, minikube) in order to test Kubernetes Client

**Reply by Ananya2001-an** [reply]
I tried running minikube but the memory requirements were not met by my pc. 
Will 'kind' work instead?

**Reply by rohanKanojia** [reply]
Kind requires Docker. So you would need to set up Docker first.

If your PC has memory requirements, maybe you can try some free remote Kubernetes cluster like [Red Hat Developer Sandbox](https://developers.redhat.com/developer-sandbox/get-started)

**Reply by Ananya2001-an** [reply]
Ok I will try with it. Thanks for the response.
